assignments/lambda-classes.js

```diff
@@ -1 +1,224 @@
 // CODE here for your Lambda Classes
+
+// ## `lambda-classes` - We need a roster of Lambda School personnel. Build it!
+
+// * We have a school to build here! This project will get you used to thinking about classes in JavaScript and building them from a brand new data set.
+// * [x] Lambda personnel can be broken down into three different types of `people`.
+//   * [x] **Instructors** - extensions of Person
+//   * [x] **Students** - extensions of Person
+//   * [x] **Project Managers** - extensions of Instructors
+// * [x] **IMPORTANT** - You'll need to create 2 - 3 objects for each class and test them according to their unique Attributes. For example:
+
+// ```js
+// const fred = new Instructor({
+//   name: 'Fred',
+//   location: 'Bedrock',
+//   age: 37,
+//   gender: 'male',
+//   favLanguage: 'JavaScript',
+//   specialty: 'Front-end',
+//   catchPhrase: `Don't forget the homies`
+// });
+// ```
+
+// #### Person
+
+// * [x] First we need a Person class. This will be our `base-class`
+// * [x] Person receives `name` `age` `location` `gender` all as props
+// * [x] Person receives `speak` as a method.
+// * [x] This method logs out a phrase `Hello my name is Fred, I am from Bedrock` where `name` and `location` are the object's own props
+
+class Person {
+    constructor(personAttr) {
+        this.name = personAttr.name;
```

Comment:
From this point on, we don't really need a unique name for the arguments in a constructor. Most of the time you'll see them called `props`.